assets/scss/modules/portfolio.scss

```diff
@@ -1,4 +1,53 @@
 .portfolio {
   background-color: $black-haze;
-  min-height: 56.25rem;
+  background: cover;
+  padding-bottom: 7.5rem;
+}
+
+.showcase {
+  padding-top: 5.625rem;
+}
+
+.img-list {
+  display: inline-block;
+  height: 260px;
+  list-style-type: none;
+  padding: 30px 10px 10px 30px;
+  width: 360px;
+}
+
+.hover-container {
+  text-align: center;
+}
+
+.work {
+  margin: 0;
+}
+/*.content {
+    background-color: $picton-blue;
+    color: $white;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    left: 0;
+    opacity: 0;
+    padding: 0 15px;
+    position: absolute;
+    text-align: center;
+    top: 0;
+    transition: opacity 500ms;
+    width: 100%;
+
+    .description {
+      flex: 1;
+    }
+}
+*/
+.img-list li:hover .content {
```

Comment:
Merge rule `.img-list li:hover .content` with rule on line 11<br>Selector should have depth of applicability no greater than 2, but was 3